dashboard/src/components/TimeRange/PrimaryRangePicker.tsx

```diff
@@ -17,13 +17,16 @@ import { DateRangeSection } from '@/components/TimeRange/DateRangeSection';
 import { getAllowedGranularities } from '@/utils/granularityRanges';
 import { formatPrimaryRangeLabel } from '@/utils/formatPrimaryRangeLabel';
 import { LiveIndicator } from '@/components/live-indicator';
+import { useDemoMode } from '@/contexts/DemoModeContextProvider';
 
 export function PrimaryRangePicker({ className = '' }: { className?: string }) {
   const [open, setOpen] = useState(false);
   const isMobile = useIsMobile();
   const t = useTranslations('components.timeRange');
```

Comment:
```suggestion
  const t = useTranslations('components');
```

I personally think its cleaner to just import the shared root path, and is more consistent with the rest of the code base.